src/test/java/testresults/PostTestResultsPozTestTypesSubmittedLvl.java

```diff
@@ -567,7 +568,7 @@ public void testResultsTestResultValueOne() {
         testResultsSteps.validateData("Test records created");
     }
 
-    @Title("CVSB-417 - CVSB-949 - CVSB-1140 / CVSB-1573 - Consumer creates a new test results for the submitted/cancelled test - testResult pass")
+        @Title("CVSB-417 - CVSB-949 - CVSB-1140 / CVSB-1573 - Consumer creates a new test results for the submitted/cancelled test - testResult pass")
```

Comment:
I think all the changes in this file can be reverted

Reply:
thanks done